pkg/utils/gitrepo.go

```diff
@@ -117,6 +122,18 @@ func CloneGitRepo(repoURL string, branch plumbing.ReferenceName, user, password,
 		}
 	}
 
+	if skipVerify {
+		// Create a custom http(s) client with your config
+		customClient := &http.Client{
+			// accept any certificate (might be useful for testing)
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			},
+		}
+		// Override http(s) default protocol to use our custom client
+		gclient.InstallProtocol("https", githttp.NewClient(customClient))
+	}
+
 	klog.Info("Cloning ", repoURL, " into ", destDir)
 	r, err := git.PlainClone(destDir, false, options)
```

Comment:
@frankwg 

can you please explain a bit in terms of how the `skipVerify` injected to the `git.PlainClone`? I'm not sure how the `githttp.NewClient(customClient)` is used by `git.PlainClone(destDir, false, options)`.

Since the `skipVerify` is not inject to the `options`

Reply:
Hi @frankwg can you please comment on the above question, before we merge this PR